ProjectSidewalk/SidewalkWebpage
# #2744: update leaderboard feet to miles cutoff

Resolves #2744

Updated the leaderboard distance cutoff. Distances will now display in miles if they are greater than 2500ft rather than 5280ft (1 mile).

Before:
<img width="1440" alt="Screen Shot 2022-01-21 at 4 21 07 PM" src="https://user-images.githubusercontent.com/50668250/150619696-5826b454-0460-46ae-b811-1d82cb451643.png">

After:
<img width="1439" alt="Screen Shot 2022-01-21 at 4 17 48 PM" src="https://user-images.githubusercontent.com/50668250/150619706-811e6719-618d-4b53-8c22-b9e2f345adfc.png">

##### Testing instructions
1. Set up your local environment with an empty database dump
2. Start server, create an account, and explore the city for a distance greater than 2500ft
3. Navigate to the leaderboard page and check that the distance displayed under your account is in miles rather than feet
